socrates/lib/activities/socratesActivityExtended.js

```diff
@@ -11,6 +11,10 @@ if (SoCraTesActivity.prototype.reserve === undefined) {
     return new SoCraTesResource(this.resourceNamed(registrationTuple.resourceName));
   };
```

Comment:
Durch die neue Methode SoCraTesActivity.prototype.socratesResourceNamed könnte diese hier zu `return this.socratesResourceNamed(registrationTuple.resourceName);` umgebaut werden